src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -446,6 +440,12 @@ class QueryCompiler(cq : ConjunctiveQuery, ss: CompilationState) {
         ${ optionalClause("WHERE", generateWhereClause(body, "")) }${groupbyStr}${limitStr}"""
   }
 
+  def generateSQL(useAlias: Boolean) = {
+    val head = generateSQLHead(useAlias)
+    val body = cq.bodies map generateSQLBody mkString("\nUNION ALL\n")
```

Comment:
I'll look into this one. Looks like a bug here. We have test cases for disjunctive bodies.